google/auth/transport/_mtls_helper.py

```diff
@@ -85,6 +87,90 @@ def _read_dca_metadata_file(metadata_path):
     return metadata
 
 
+def _get_workload_x509(certificate_config_path=None):
+    """Read the cert and key files specified in the certificate config provided.
+    If no config path is provided, check the environment variable and well known
+    gcloud location.
+
+    Args:
+        certificate_config_path (string): The certificate config path. If no path is provided,
+        the well known gcloud location will be used.
+
+    Returns:
+        Tuple[bytes, bytes]: client certificate bytes in PEM format and key
+            bytes in PEM format.
+
+    Raises:
+        google.auth.exceptions.ClientCertError: if problems occurs when retrieving
+        the certificate or key information.
+    """
+    absolute_path = _get_cert_config_path(certificate_config_path)
+    if absolute_path is None:
+        return None, None
```

Comment:
Should this be an exception?

Reply:
I don't think we want to throw here, since generally if we don't find a valid file we would want the calling method to just call the next certificate method (i.e. ECP/secure connect) - will revisit when this is actually called as part of the cert logic